svgdigitizer/svgplot.py

```diff
@@ -131,14 +132,29 @@
 #  You should have received a copy of the GNU General Public License
 #  along with svgdigitizer. If not, see <https://www.gnu.org/licenses/>.
 # ********************************************************************
+
 import logging
+from enum import Enum
 from functools import cache
+from itertools import groupby
 
 import pandas as pd
 
 logger = logging.getLogger("svgplot")
 
 
+class PageOrientation(Enum):
+    r"""Enum representing the page orientation to be able to to autodetect axis orientation."""
+    PORTRAIT = "portrait"
+    LANDSCAPE = "landscape"
+
+
+class AxisOrientation(Enum):
+    r"""Enum representing axis orientation."""
```

Comment:
```suggestion
    r"""
    The orientation of a plot axis.
    """ 
```

Just trying to rewrite this so it does not exactly replicate what is written in the very line above it.